mmv1/third_party/terraform/tests/resource_gkeonprem_bare_metal_cluster_test.go.erb

```diff
@@ -0,0 +1,403 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccGkeonpremBareMetalCluster_bareMetalClusterUpdateBasic(t *testing.T) {
+  // VCR fails to handle batched project services
```

Comment:
Note: Same as in https://github.com/GoogleCloudPlatform/magic-modules/pull/7824 we are gonna want to remove these.

Reply:
Removed. Thanks for the correction!